docs/howto/kubernetes-large-clusters.md

```diff
@@ -7,7 +7,7 @@ Starting from AKS Engine v0.3.0, AKS Engine supports using exponential cloud bac
 ## To Use
 
 Declare your kubernetes cluster API model config as you normally would, with the following requirements:
-- You must be using at minimum the `v1.6.6` version of Kubernetes to have access to the `kubernetesConfig` configuration vectors exemplified in [examples/largeclusters/kubernetes.json](https://github.com/Azure/aks-engine/blob/master/examples/largeclusters/kubernetes.json). As long as you are using a version of AKS Engine `v0.3.0` or newer, your kubernetes cluster specification will fulfill this minimum version requirement.
+- You must be using at minimum the `v1.12.8` version of Kubernetes to have access to the `kubernetesConfig` configuration vectors exemplified in [examples/largeclusters/kubernetes.json](https://github.com/Azure/aks-engine/blob/master/examples/largeclusters/kubernetes.json). As long as you are using a version of AKS Engine `v0.3.0` or newer, your kubernetes cluster specification will fulfill this minimum version requirement.
```

Comment:
Or as I said above, since this is talking about "minimum version v1.6.6," it's not relevant for aks-engine users any more and this entire bullet point can be removed.